app-shell/scripts/before-build.js

```diff
@@ -88,10 +90,13 @@ module.exports = function beforeBuild(context) {
         `--target=${path.join(PYTHON_DESTINATION, sitePackages)}`,
         path.join(__dirname, '../../shared-data/python'),
         path.join(__dirname, '../../api'),
+        'pandas==0.25.3',
```

Comment:
Can we leave a comment like this?

```suggestion
        'pandas==0.25.3', // To match what's installed on robots.
```